### PR TITLE
added build and integration-test commands in readme

### PR DESCRIPTION
**Pull request by naraharirohith**
[AR-3292](https://team-1624093970686.atlassian.net/browse/AR-3292)

SDK Readme references for build and integration tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,21 @@ You can use the standalone module which includes the polyfills.
...
 
+# :computer: Development
+## Build
+```
+npm run build
+```
+
+## Integration testing
+
+Run an HTTP server from the storage SDK, and go to */integration-tests/*. Connect to metamask with arcana beta RPC (https://blockchain001-testnet.arcana.network/). Make sure that you are using the correct chain-Id (40405) and gateway URL (https://gateway001-testnet.arcana.network/) in the */integration-tets/test.js/* while initializing the storage instance.
```
[the remark this answers is not in the document]
Comment (shaloo) [reply README.md:62]: 1. I dont understand what is 'RUn an HTTP server from the Storage SDK'?  Did you mean clone sdk, modify, build and then run something to test? Pls list exact instruction that say brings up localhost:xyz port for example
2. Change directory to */integration-tests/* folder.
3. In the */integration-tets/test.js file, make sure you are using correct chian-ID (40405) and gateway URL https://gateway001-testnet.arcana.network/ while initializing Arcana Storage SDK instance.
3. Configure your Wallet: A.) Metamask a) Use browser and connect to metamask wallet b.) Configure Metamask for Arcana Network and specify RPC URL as https://blockchain001-testnet.arcana.network/. B.) Arcana Auth (Wallet) - is there any setup or testing done using our own wallet? How to set that up should we point to arcana docs?

[the remark this answers is not in the document]
Comment (shaloo) [reply README.md:62]: Ive tried to clearly separate out steps that need to be done in Storage SDK code test init vs metamask wallet config.  Also, for testing is metamask teh only way? what if a developer is using arcana auth? how do they test it?

[the remark this answers is not in the document]
Comment (naraharirohith) [reply README.md:62]: - We can run http server using `http-server`.
- Not quite sure whether we can use arcana wallet or not.

So, imk the steps will be,
1. In the */integration-tets/test.js file, make sure you are using the correct chain-ID (40405) and gateway URL https://gateway001-testnet.arcana.network/ while initializing the Arcana Storage SDK instance.
2. Start running a HTTP server by `http-server`.
3. Change directory to /integration-tests/ folder.
4. Make sure to configure metamask for Arcana Network and specify RPC URL as https://blockchain001-testnet.arcana.network/.

[the remark this answers is not in the document]
Comment (SauravKanchan) [reply README.md:62]: There are many ways to run http server

+ Eg for python 3
```
python3 -m http.server
```

+ Eg for python
```
python -m SimpleHTTPServer
```

+ One can also use packages like [http-server](https://www.npmjs.com/package/http-server), [browser-sync](https://browsersync.io/)

**Review by SauravKanchan — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -74,6 +75,34 @@ For any support or integration related queries, contact [Arcana support team](ma
...
+## Build
+```
+npm run build
+```
+
+## Integration testing
+
+1. In the */integration-tets/test.js* file, make sure you are using the correct chain-ID (40405) and gateway URL (https://gateway001-testnet.arcana.network/) while initializing the Arcana Storage SDK instance.
+
+2. Start running a HTTP server by http-server. There are many ways to run http server
```
Comment (shaloo) [inline README.md:89]:
```suggestion
Start running an HTTP server using one of these suggested methods:

**Review by shaloo — APPROVED** [review]
I have modified one line and committed to the readme directly. If you are ok with that - we are good to go. Thx.

**Pull request merged**